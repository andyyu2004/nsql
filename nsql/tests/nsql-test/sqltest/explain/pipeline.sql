SET explain_output = 'pipeline';

CREATE TABLE t (
    id int PRIMARY KEY,
    b boolean
);

EXPLAIN UPDATE
    t
SET
    b = TRUE
WHERE
    b;
