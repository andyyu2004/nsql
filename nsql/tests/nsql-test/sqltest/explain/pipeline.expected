 metapipeline #0                                                   
   pipeline #0                                                     
      node #4 output                                               
      node #3 update t                                             
                                                                   
       metapipeline #1                                             
         pipeline #1                                               
            node #3 update t                                       
            node #2 projection (:project @0 :ret, :push true :ret) 
            node #1 filter :project @1 :ret                        
            node #0 scan t (id, b)                                 
                                                                   